src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,17 @@
+@mixin text-main {
+  font-size: 12px;
+  line-height: 18px;
+}
+
+@mixin text-secondary {
+  font-size: 10px;
+  line-height: 14px;
+  font-weight: 400;
+  color: $secondary-text-color;
```

Comment:
According to the code style rules, variables for colors should be used in a way that describes their purpose rather than their content. For example, instead of `$secondary-text-color`, you might use a name like `$text-color-muted` or `$text-color-secondary` if this color is specifically for secondary text across the site. This makes it easier to understand the role of the color in the design system and maintain the codebase.